sc-tools/sc-builder/src/sc_repo_path_collector.cpp

```diff
@@ -0,0 +1,180 @@
+/*
+ * This source file is part of an OSTIS project. For the latest info, see http://ostis.net
+ * Distributed under the MIT License
+ * (See accompanying file COPYING.MIT or copy at http://opensource.org/licenses/MIT)
+ */
+
+#include "sc_repo_path_collector.hpp"
+
+#include <regex>
+#include <filesystem>
+
+#include "sc-memory/sc_utils.hpp"
+
+std::unordered_set<std::string> const ScRepoPathCollector::m_supportedSourcesFormats = {"scs", "gwf"};
+std::unordered_set<std::string> const ScRepoPathCollector::m_supportedRepoPathFormats = {"path"};
+
+namespace impl
```

Comment:
why do you need this namespace

Reply:
To private implementation. It must be used only in this file

Reply:
make private method